### ISSUE TITLE
Android Issue

### ISSUE THREAD

**Issue by devz3ro** [jumpmanjay/HDGrandSlam.bundle#5]
First of all great project, your hard work is definitely appreciated.

I have been using the following plex plugin to watch TV remotely until I came across this plugin.
https://github.com/zynine-/HDHRViewerV2.bundle

I just tried this plugin and I unfortunately have the same exact issue with it as I do with HDHRViewerV2. Not sure if this can be fixed but I figured it wouldn't hurt to ask.

What I'm running:
HDHomeRun: HDTC-2US 20161012beta1
Plex Media Server: plexmediaserver-1.2.3.2914-1ff0f18 (plex pass)

The Problem:
I have a Android TV (Google ADT-1), and this plugin only works when I set Passthrough audio to auto, and when I create a Profiles directory in the PMS data root, and put in a Android.xml file with the following code: https://github.com/zynine-/PlexDeinterlaceProfiles/blob/master/Profiles/Android.xml

Unfortunately, while the Android.xml allows playback for this plugin (HDGrandSlam) as well as the HDHRViewerV2, it breaks the playback of all my other media :-/.

I've played around with the XML in the Androd.xml file. I made sure to restart the PMS process every time during my testing to ensure it was reloaded. Unfortunately my local media only plays correctly when `audioCodec="aac"` on the `protocol="hls"` line. I changed the container to "mkv" and tried audioCodec: he-aac, ac3, eac3, mp2, mp3, vorbis, flac, alac, wma, and pcm. The only codecs that don't error out are flac and alac, but unfortunately they have no sound.

In short: My local media will only play (with sound) when `audioCodec="aac"`, and the HDGrandSlam plugin will only play when `audioCodec="anythingbutaac"`. This is inside the Android.xml file on the `protocol="hls"` line in the Profiles directory.

I'm looking for a happy medium.

**Comment by devz3ro** [comment]
This was fixed using the following xml:

https://github.com/zynine-/HDHRViewerV2.bundle/files/554533/Android.zip

**Comment by nicktones** [comment]
Close?
